### ISSUE TITLE
[Bug] Upload to hidden input doesn't work

### ISSUE THREAD

**Issue by daniula** [abramenal/cypress-file-upload#41]
### Current behavior:

Uploading file to a hidden input ends up with below error:

> CypressError: Timed out retrying: cy.trigger() failed because this element is not visible:
> 
> <input id="image-upload" accept="image/*" type="file" multiple="" autocomplete="off" tabindex="-1" style="display: none;">
> 
> This element '<input#image-upload>' is not visible because it has CSS property: 'display: none'
>
> Fix this problem, or use {force: true} to disable error checking.
>
> https://on.cypress.io/element-cannot-be-interacted-with

There is no way to pass option `{ force: true }` as an error message suggests. Current workaround is calling `.invoke('show')` before calling `.upload()`:

```js
    cy.fixture('avatar.jpg').then((fileContent) => {
      cy.get('=image-upload input')
        .invoke('show')
        .upload(
          { fileContent, mimeType: 'image/jpeg' },
          { subjectType: 'input' }
        )
    })
```

### Desired behaviour:

`.upload` command accepts `force` option:

```js
    cy.fixture('avatar.jpg').then((fileContent) => {
      cy.get('=image-upload input')
        .upload(
          { fileContent, mimeType: 'image/jpeg' },
          { subjectType: 'input', force: true }
        )
    })
```

### Versions

* cypress: 3.2.0
* cypress-file-upload:  3.0.4
* operating system: macOS 10.14.1

**Comment by abramenal** [comment]
Hi @daniula 
Thanks for your contribution.

Just released v3.0.5!
Check out the new API and re-open issue in case of any further issues.

@allcontributors[bot] add @daniula for question

**Comment by allanslim** [comment]
Hi Everyone,
I am getting this error even with `force: true`. Does `force:true` only works for input and not in drag-and-drop? I also tried `Invoke('show')` but it didn't help either. The way our upload works, is that the DropZone div disappears after it get's the file, and a button ( "Upload" ) needs to be clicked to upload the file.

```
CypressError: cy.trigger() failed because this element is detached from the DOM.

<div data-cy="dropzone">...</div>

Cypress requires elements be attached in the DOM to interact with them.

The previous command that ran was:

  > cy.trigger()

This DOM element likely became detached somewhere between the previous and current command.

Common situations why this happens:
  - Your JS framework re-rendered asynchronously
  - Your app code reacted to an event firing and removed the element
```
Any help will be greatly appreciated. Here are the versions:

cypress-file-upload: ^3.5.3
 cypress:  ^3.8.3
MacOS: 10.15.2

Thank you!
